src/components/programmingCard.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react"
+import { Link } from "gatsby"
+import "./programmingCard.scss"
+
+export default function ProgrammingCard({ title, summary, id, entries }) {
+  const href = `https://www.artsmatter.org/programs/${id}`
```

Comment:
@bradybridges just a follow up on making this `href` relative to the current domain, so that we can use it on our local dev and staging as well.